kythe/go/serving/tools/write_tables/write_tables.go

```diff
@@ -64,17 +67,32 @@ var (
 
 	experimentalBeamPipeline = flag.Bool("experimental_beam_pipeline", false, "Whether to use the Beam experimental pipeline implementation")
 	beamShards               = flag.Int("beam_shards", 0, "Number of shards for beam processing. If non-positive, a reasonable default will be chosen.")
+	beamK                    = flag.Int("beam_k", 0, "Amount of memory to use when creating level DB shards.")
```

Comment:
Describe units and default

Reply:
Updated the string. Let me know if this makes sense.